Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM botpress/server:v12_5_0
+FROM botpress/server:v12_23_0
```

Comment:
Probably we can put `latest`. The first release tag will be tonight. So we can merge this PR really quickly before the official release.